src/main/java/org/usfirst/frc4904/robot/commands/ExtendIntake.java

```diff
@@ -0,0 +1,16 @@
+package org.usfirst.frc4904.robot.commands;
+
+import org.usfirst.frc4904.robot.subsystems.Intake;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class ExtendIntake extends CommandBase {
+    protected final Intake intake;
+    protected double targetSpeed;
+    public ExtendIntake(Intake intake, double targetSpeed) {
```

Comment:
The intake should not be a parameter of the constructor. Instead, use `this.intake = RobotMap.Component.intake;`. Additionally, for this to work, add the line `addRequirements(RobotMap.Component.intake);` as the last line of the constructor. You can checkout to the turret branch and look at the command `TurnTurret` as an example.